account_move_service_report/report/report_sale_xlsx.py

```diff
@@ -0,0 +1,72 @@
+from odoo import _, models
+
+
+class AbstractReportXslx(models.AbstractModel):
+    _name = "report.account_move_service_report.report_sale_xlsx"
+    _description = "Abstract XLSX Account Move Service Report"
+    _inherit = "report.report_xlsx.abstract"
+
+    def report_values(self):
+        return {
+            _("Provider name"): ["company_id", "name"],
+            _("ID Number (8 digits)"): ["auth_number"],
+            _("Service Type"): [lambda x: x.get_service_typology_name()],
+            _("Service Date"): ["service_date", lambda x: x.date()],
+            _("Service Time"): ["service_date", lambda x: x.time()],
+            _("Insurance Name"): ["service_name"],
+            _("Insurance Code"): ["service_code"],
+            _("Origin"): ["origin"],
+            _("Destination"): ["destination"],
+            _("KM Quantity"): [lambda x: x.get_service_type_quantity("km")],
+            _("€/KM"): [lambda x: x.get_service_type_weighted_average_price("km")],
+            _("Total KM (€)"): [lambda x: x.get_service_type_amount_total("km")],
+            _("DEPARTURE (€)"): [lambda x: x.get_service_return_price_subtotal(False)],
+            _("RETURN (€)"): [lambda x: x.get_service_return_price_subtotal(True)],
+            _("ADDITIONAL (concept)"): [lambda x: x.get_service_additional_concept()],
+            _("ADDITIONAL (€)"): [lambda x: x.get_service_type_subtotal("additional")],
+            _("WAITING TIME (€)"): [lambda x: x.get_service_type_subtotal("wait")],
+            _("Total by ID (€)"): [lambda x: x.get_service_total_by("auth_number")],
+            _("Total Insurance (€)"): [
+                lambda x: x.get_service_total_by("service_code")
+            ],
+            _("Patient (name and surname)"): ["insured_name"],
+            _("Invoice Date"): ["service_invoice_creation_date"],
+            _("Invoice Number"): ["service_invoice_number"],
```

Comment:
Get these fields from `account_move` model directly, these

```suggestion
            _("Invoice Date"): ["date"],
            _("Invoice Number"): ["name"],
```

Reply:
It's invoice_date instead of date, right?
![image](https://github.com/user-attachments/assets/e129f951-fb8a-4178-b7ff-ae6e966e4f7c)